resources/default_pipeline_environment.yml

```diff
@@ -367,7 +367,7 @@ steps:
     stepMessage: 'Do you want to restart?'
   pipelineStashFilesAfterBuild:
     stashIncludes:
-      buildResult: '**/target/*.war, **/target/*.jar, **/*.mtar, **/*.jar.original'
+      buildResult: '**/target/*.war, **/target/*.jar, **/*.mtar, **/*.jar.original, dist/**'
```

Comment:
for which uses cases, dist/** is needed?

Reply:
AFAIK, for npm builds and uploading to cts.

Reply:
use **/dist/** instead. the source root could be changed.

Reply:
At the moment this folder is created on the root as mentioned in the [docu](https://github.com/SAP/jenkins-library/blob/master/documentation/docs/steps/transportRequestUploadFile.md#cts-uploads). Setting `**/dist/**` would unnecessarily stash other items which might not be of interest. Do you still think we need to set like `**/dist/**`?

Reply:
yes, you are right. if the source root is changed, even the folder name would be changed. so the wildcard doesn't make much sense.